src/tools/kdb/import.cpp

```diff
@@ -63,16 +63,9 @@ int ImportCommand::execute(Cmdline const& cl)
 
 	KeySet base;
 
-	// TODO: this should not be neccessary, but
-	// applying threeway strategies to a twoway merge is hard
-	base = KeySet();
+	base = existingKeys;
```

Comment:
useless copy: directly use existingKeys (you may rename it) or use KeySet const& base